### ISSUE TITLE
[Chore] Tar file is being published to npm

### ISSUE THREAD

**Issue by eligolding** [mobi/goponents#51]
When installing this module from npm, there is a tarball being installed along with it, which is really not necessary.

<img width="400" alt="index d ts 2019-07-02 15-15-44" src="https://user-images.githubusercontent.com/5261826/60541950-ac5e9d80-9ce0-11e9-91c1-be1871fa2684.png">

It seems like this is created by `npm pack` which is part of the [publish script](https://github.com/mobi/goponents/blob/256fc91cfb019c3377d651268db6f3f3b18e4d30/package.json#L24) of this lib. I believe the publish script can simply build and then publish like so:
```
"publish": "npm run build_lib && cd dist/go-lib && npm publish",
```

If I'm missing something here and this is really correct behavior feel free to close. Otherwise I'll be more than happy to open a PR for this.

**Comment by AlexOverbeck** [comment]
Thanks @eligolding! We will do some research here to make sure things are getting published properly and roll a solution into a subsequent version.

**Comment by grahamhency** [comment]
Closed with #555!
